MetaMask/metamask-extension
# fix: Sort token order by fiat amount (highest => lowest)

## Explanation

Sort token order by fiat amount (highest => lowest)


Description

We currently don’t have a intuitive sorting of tokens. We should be sorting by token balances (by $).


## Screenshots/Screencaps

### Before
<img width="1381" alt="Screenshot 2023-09-07 at 14 40 09" src="https://github.com/MetaMask/metamask-extension/assets/26223211/c0460334-d39e-4d17-88d9-79460c07860e">

### After

<img width="1475" alt="Screenshot 2023-09-07 at 14 39 34" src="https://github.com/MetaMask/metamask-extension/assets/26223211/f566cb4a-8746-44d2-8012-820f2d0c7bcf">


## Manual Testing Steps

- Open Metamask extension
- go to the tokens tab
- the list should be sorted by fiat from highest to lowest
